release-notes/Code_Libraries/Skyline.DataMiner.DataSources.OpenConfig/Skyline_DataMiner_DataSources_OpenConfig_2.0.0.md

```diff
@@ -0,0 +1,17 @@
+---
+uid: Skyline_DataMiner_DataSources_OpenConfig_2.0.0
+---
+
+# Skyline DataMiner DataSources OpenConfig 2.0.0
+
+## New features
+
+#### Improvements related to data grids [ID_37758]
+
+Several improvements have been implemented related to connector data grids:
+
+- Depending on the vendor, a YANG model could be added as a prefix to the name of a node leaf or not. Up to now, the DataMinerConnectorDataGridColumn needed to have the YANG model as prefix in the path to be able to support both situations. This is now no longer required. For example, in the past `openconfig-interfaces:admin-status` had to be defined, while now it will suffice to define `admin-status`.
```

Comment:
Could the following be added: "However, we still strongly recommended adding the YANG module name as a prefix to have a unique identifier when a value needs to be mapped with a column. "? This is the same line that is mentioned in the "important" section that is documented under https://docs.dataminer.services/develop/DSI/DSIOpenConfig/DSI_OpenConfig_Middleware.html#mapping-a-yang-path-to-a-parameter .

Reply:
I've added this as a note.